packages/core/strapi/tests/api/relations.test.api.js

```diff
@@ -687,4 +768,60 @@ describe('Relations', () => {
       });
     });
   });
+
+  // TODO: Use createShop and updateShop in other tests
+  describe.only('Reorder entity relations', () => {
+    test('Connect new relation at the start', async () => {
+      // Add two products at the start
+      const createdShop = await createShop({
+        anyToManyRel: [
+          { id: id1, position: { start: true } },
+          { id: id2, position: { start: true } },
```

Comment:
add another test case to test it is reordering when creating a new entity, and change start:true to before: id1